src/main/java/bio/terra/app/controller/DatasetsApiController.java

```diff
@@ -222,6 +247,17 @@ public ResponseEntity<ColumnStatisticsModel> lookupDatasetColumnStatisticsById(
     return ResponseEntity.ok(columnStatisticsModel);
   }
 
+  @Override
+  public ResponseEntity<ColumnStatisticsModel> queryDatasetColumnStatisticsById(
```

Comment:
Do we also want to mark `lookupDatasetColumnStatisticsById` as `@Hidden`?

Reply:
Would you also mind speaking to why we need to mark these endpoints as deprecated in the Swagger spec as well as annotating them with `@Hidden` in the implementation?

Reply:
Oh, thanks for catching that. 
I didn't realize it was missing because it actually looks like the `@Hidden` doesn't change the swagger UI treatment. I thought that marking it as hidden was why these endpoints were crossed out in the swagger ui. But, it appears that's just done by marking the endpoint as deprecated.  
So, I think the hidden annotation isn't working as expected. I think it's supposed to actually completely hide it from the swagger ui (not just cross it out). But, I think we're okay with just the deprecated behavior. I'll just remove the hidden annotation.